### ISSUE TITLE
pytest warning for yaml parser

### ISSUE THREAD

**Issue by hugoledoux** [tudelft3d/val3dity#142]
```
================================================ warnings summary =================================================
tests/test_metadata.py::test_metadata
  /Users/hugo/projects/val3dity/tests/test_metadata.py:14: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
    ds = yaml.load(stream)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
```

**Comment by balazsdukai** [comment]
Which pyyaml version are you using?

**Comment by balazsdukai** [comment]
Fixed in bdb01c8cd39b54d562fad0b628a75d2202007398
